### PR TITLE
blog: add the v0.13.0 release post

### PR DESCRIPTION
**Pull request by rich-iannone**
This adds the blog post for the v0.13.0 release of the package.

### CODE REVIEW

File: docs/blog/introduction-0.13.0/index.qmd [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+
+### The *combined* location helpers: `loc.column_header()`, `loc.footer()`, and `loc.header()`
+
+Look, I know we brought up the expression *fine-grained* before---right in the first paragraph---but sometimes you need just the opposite. There are lots of little locations in a GT table and some make for logical groupings. To that end, we have the concept of *combined* location helpers. There are three of these and they equate to some collection of location helpers:
+
+- `loc.header()` == `[loc.title(), loc.subtitle()]`
+- `loc.column_header()` == `[loc.column_labels(), loc.spanner_labels()]`
+- `loc.footer()` == `[loc.source_notes()]`
+
+That last one, `loc.footer()`, is currently just an alias for `loc.source_notes()`. However, in the future, when footnotes support is available in the package, the set will expand to include a `loc.footnotes()` location.
```
Comment (machow) [inline docs/blog/introduction-0.13.0/index.qmd:179]: It felt a bit confusing to see details on three combined selectors, but only two used in the example. It seems okay to mention the gist of combined selectors, with a tiny explanation after the example (can link to any relevant docs for a full enumeration)

```suggestion
Look, I know we brought up the expression *fine-grained* before---right in the first paragraph---but sometimes you need just the opposite. There are lots of little locations in a GT table and some make for logical groupings. To that end, we have the concept of *combined* location helpers.

```

File: docs/blog/introduction-0.13.0/index.qmd [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+    gt_tbl
+    .tab_style(
+        style=[style.text(v_align="middle"), style.fill(color="#EEEEEE")],
+        locations=[loc.stubhead(), loc.column_header(), loc.footer()]
+    )
+)
+
+gt_tbl
+```
+
```
Comment (machow) [inline docs/blog/introduction-0.13.0/index.qmd:194]: Could note that `column_header()` targets both loc.column_labels() and loc.spanner_labels(), and then link to composite selector page etc...

File: docs/blog/introduction-0.13.0/index.qmd [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+Though the order of things matters when setting styles via `tab_style()`, it's not a problem here to set a style for the combined 'header' location after doing so for the 'title' and 'subtitle' locations because the 'font' attribute *wasn't* set by `tab_style()` for those smaller locations.
+
+### How `tab_style()` fits in with `tab_options()`
+
+When it comes to styling, you can use `tab_options()` for some of the basics and use `tab_style()` for the more demanding styling tasks. And you could combine the usage of both in your table. The precedence (or priority) is given to `tab_style()`, regardless of the order of invocation. As an example of this, we can add a background color to all remaining locations with `tab_options(table_background_color=...)`:
```
Comment (machow) [inline docs/blog/introduction-0.13.0/index.qmd:256]: I would move this to after the example, with a short sentence explaining to people what is changing in the table (e.g. HoneyDew fill)

**Review by machow — CHANGES_REQUESTED** [review]
This is looking great, and the example legit helped me wrap my head around all the loc specifiers!

The lead-in to examples feels like it might be a bit too long, and do a lot of factual and procedural breakdowns / enumeration beforehand. I left suggestions to cut down and move some of this to after examples. Feel free to use, tweak, and punt to your heart's delight! (I'm okay with us posting any version of this including as is)

File: docs/blog/introduction-0.13.0/index.qmd [hunk 4]
```diff
@@ -0,0 +1,270 @@
+---
+title: "Great Tables `v0.13.0`: Styles all over the table"
```
[the remark this answers is not in the document]
Comment (rich-iannone) [reply docs/blog/introduction-0.13.0/index.qmd:2]: I went went the second one, much better!

File: docs/blog/introduction-0.13.0/index.qmd [hunk 5]
```diff
@@ -0,0 +1,270 @@
...
+    .opt_align_table_header(align="left")
+    .opt_table_font(font=google_font(name="IBM Plex Sans"))
+    .opt_vertical_padding(scale=0.5)
+    .opt_horizontal_padding(scale=2)
+)
+
+gt_tbl
+```
+
+This table will serve as a great starting point for demonstrating all the things you can do with `tab_style()`. (It'll be `tab_style()` like you've never seen it before!)
```
[the remark this answers is not in the document]
Comment (rich-iannone) [reply docs/blog/introduction-0.13.0/index.qmd]: Thanks this is great. I took this suggestion and created a basic styling plan based on these bullet points. Much better now!

File: docs/blog/introduction-0.13.0/index.qmd [hunk 6]
```diff
@@ -0,0 +1,270 @@
...
+---
+
+We did something in Great Tables (`0.13.0`) that'll make your tables that much more customizable: super *fine-grained* ways of setting styles throughout the table. Before you were largely constrained to styling through the following strategies:
+
+1. use a limited set of styles (e.g., background color, font weight, etc.) to different table locations like the stub, the column labels, etc., through `tab_options()`
+2. use `tab_style()` with a larger set of styling options for the table body cells (specified by `loc.body()`)
+
+In `v0.13.0`, we can target much more than just the table body! Here is the expanded set of `loc.*()` methods along with the locations that they can target.
+
+- `loc.title()`: the table title
```
[the remark this answers is not in the document]
Comment (rich-iannone) [reply docs/blog/introduction-0.13.0/index.qmd:17]: Excellent idea and much more direct. I will work on this and get it in soon!

File: docs/blog/introduction-0.13.0/index.qmd (hunk 2 above)
Comment (rich-iannone) [reply docs/blog/introduction-0.13.0/index.qmd:196]: Done!

File: docs/blog/introduction-0.13.0/index.qmd (hunk 6 above)
[the remark this answers is not in the document]
Comment (rich-iannone) [reply docs/blog/introduction-0.13.0/index.qmd]: This is now in.

File: docs/blog/introduction-0.13.0/index.qmd (hunk 3 above)
Comment (rich-iannone) [reply docs/blog/introduction-0.13.0/index.qmd]: This has been rewritten with the explanatory sentences after the example code.

**Review by machow — CHANGES_REQUESTED** [review]
It's looking really great! One more quick nit, if it's okay. I noticed leading into a few examples what people should see in the output is not explained.

For example...

* "you can do this:" (what is this?)
* "let's style" (what style should I look for?)
* "add an assortment of styles to it." (what styles? what should I look for?)
* "let's use tab_options() here" (to do what? what should I look for?)

Could we tell people specifically what will change (so they know what to look for)? E.g.

* "You can manually indent STABLE using a CSS string:"
* "Let's set a grey background fill on the column header, stubhead, and footnotes:" (no need to say you are using `tab_style()` people will see it, and they have seen `tab_style()` used several times at this point.)
* Set an aliceblue background fill on the title, with some font and border adjustments.
* Set a default honeydew background fill on body values.

(Definitely tweak as needed! I think the key is that people should be able to quickly identify what in the table will change; it's okay to just say the easiest to spot piece.)

### Screenshots

image


image


image


image

File: docs/blog/introduction-0.13.0/index.qmd [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+### Don't forget the stub!
+
+We mustn't forget the stub. It's a totally separate location, being off to the side and having the important responsibility of holding the row labels. Here, we are going to do two things:
+
+1. Change the fill color (to 'Linen') and make the text bold for the *entire stub*
+2. Highlight the rows where we have stable isotopes (the extent is both for the stub and the body cells)
+
+For task #1, a simple `loc.stub()` call in `tab_style()`'s `locations=` argument will be enough to target the entire stub.
+
+Task #2 is more interesting. Like `loc.body()`, `loc.stub()` has a `rows=` argument. Because of that, we can use the same Polars expression to target those rows that belong to the stable isotopes (they are identifiable by having `None` values in the `half_life` column). Once we know about the power of Polars expressions in `rows=`, we realize just how much power we have in targeting our table stylings.
```
[the remark this answers is not in the document]
Comment (machow) [reply docs/blog/introduction-0.13.0/index.qmd:132]: I think the last piece might just be removing this, since it's down below the example now!

**Review by machow — APPROVED** [review]
Yo, this looks great! I left a small note on a paragraph that's duplicated above and below an example now. It looks like the top piece just needs to be cut out

**Pull request merged**